src/main/java/uk/gov/hmcts/reform/unspec/validation/interfaces/ParticularsOfClaimValidator.java

```diff
@@ -17,4 +17,13 @@ default CallbackResponse validateParticularsOfClaim(CallbackParams callbackParam
             .errors(servedDocumentFiles.getErrors())
             .build();
     }
+
+    default CallbackResponse validateParticularsOfClaimBackwardsCompatible(CallbackParams callbackParams) {
+        ServedDocumentFiles servedDocumentFiles = ofNullable(callbackParams.getCaseData().getServedDocumentFiles())
+            .orElse(ServedDocumentFiles.builder().build());
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .errors(servedDocumentFiles.getErrors())
```

Comment:
if this is backwards compatible method, then it should call `getErrorsBackwardsCompatible` here